content/api/rest/quickstart.md

```diff
@@ -12,170 +12,142 @@ sidebar: 'api-reference'
 
 This is a quick start guide to working with the [appbase.io REST API](https://rest.appbase.io).
 
-## Creating an App
+## Create Cluster
 
-This gif shows how to create an app on appbase.io, which we will need for this quickstart guide.
+You can run dedicated Elasticsearch clusters with Appbase.io services to meet your search requirements using [Appbase.io clusters](/docs/hosting/clusters/).
```

Comment:
@lakhansamani This needs an edit imo. We should cover all options. Text can be:

You can start by creating an [Elasticsearch cluster with appbase.io](href1) or [bring your Elasticsearch cluster](href2) or [self-host appbase.io](href3).

Reply:
hrefs need to point to the right pages.